### ISSUE TITLE
Custom dataset HOW-TO and is this capable of conditional GAN ?

### ISSUE THREAD

**Issue by thusinh1969** [VITA-Group/TransGAN#43]
I have 2 questions:

1) Can you please share how to train custom dataset (structure of data path) ?
2) Is this model capable to train conditional GAN ?

Thanks,
Steve

**Comment by yueruchen** [comment]
Hi, thanks for your interests.
1. Custom dataset will be very easy, you can just use celeba dataloader and save your data in a specific directory.
2. Currently it does not support conditional GAN, I would suggest you to consider some conditional normalization layer and projection discriminator, as suggested by BigGAN.

**Comment by meh-naz** [comment]
I cannot seem to figure out,
1. How to test using cifar_checkpoint on my own dataset
2. How can I train this model on my own dataset (can it be paired or unpaired?) and how can I test my trained model on my own dataset?
Please help.

**Comment by yueruchen** [comment]
@meh-naz 
1. This is a GAN model. cifar_checkpoint onl works for cifar
2. For training on your on dataset, you need to write custom dataloader and keep the other part same.

**Comment by mexicantexan** [comment]
@yifanjiang19 just for reference to make sure I have it set up properly, what would the training dataset directory structure look like?

**Comment by yifanjiang19** [comment]
just put all image in the folder

**Comment by sll0701** [comment]
Hello, can you share how to train a custom dataset? What code needs to be modified to successfully train the custom data set, I really need your help, thank you
